frontend/src/bundles/common/components/player/player.tsx

```diff
@@ -36,12 +43,13 @@ const Player = ({
                 compositionWidth={size.width}
                 compositionHeight={size.height}
                 fps={30}
-                style={{
-                    height: '100%',
-                    maxWidth: '100%',
-                }}
+                style={
+                    orientation === VideoPreview.LANDSCAPE
+                        ? LandscapeStyle
+                        : PortraitStyle
+                }
```

Comment:
Can we replace it with className and instead of creating ts file with styles use css modules?